TappNetwork/home-visiting-app-web
# Cu 25tmtn9 edit account

# Details

Home Visitors are now able to edit their profile from the web app. Edit page uses a search multi select component built using ionic components which I plan on turning into an npm package. 

# Steps to test changes

1. Pull down branch in both repos
2. run testing suite on back end repo
3. grab visitor credentials from db and login to front end app
4. navigate to user profile using the profile icon in the sidebar
5. click edit profile button
6. make changes to all fields including multi selects
7. click submit changes
8. go back to profile and confirm changes. May take an extra load for vuex to catch up
9. Navigate to profile edit page several times, ensuring that options for multi select are properly fetched each time

# Issues Addressed

[25tmtn9](https://app.clickup.com/t/25tmtn9)

# Screenshots

<img width="1360" alt="Screen Shot 2022-08-11 at 9 50 13 AM" src="https://user-images.githubusercontent.com/29612767/184175686-05f336b1-96f1-49f9-95d2-37acd8fbf943.png">
